### ISSUE TITLE
[Team management] Ungate team management to show to all users

### ISSUE THREAD

**Issue by terryttsai** [Recidiviz/justice-counts#305]
@terryttsai commented on [Fri Jan 13 2023](https://github.com/Recidiviz/recidiviz-data/issues/17878)

Once team management is ready, make it available to all users to see in Agency Settings.

**Comment by lilidworkin** [comment]
@terryttsai if you could knock this out (if it's not already done) that would be great!
